site/documentation/content/getting-started/_index.md

```diff
@@ -189,10 +189,10 @@ echo "export HTTP_ADAPTER_IP=$(kubectl get service eclipse-hono-adapter-http --o
 echo "export MQTT_ADAPTER_IP=$(kubectl get service eclipse-hono-adapter-mqtt --output="jsonpath={.status.loadBalancer.ingress[0]['hostname','ip']}" -n hono)" >> hono.env
 KAFKA_IP=$(kubectl get service eclipse-hono-kafka-0-external --output="jsonpath={.status.loadBalancer.ingress[0]['hostname','ip']}" -n hono)
 TRUSTSTORE_PATH=/tmp/truststore.pem
-kubectl get secrets eclipse-hono-kafka-example-keys --template="{{index .data \"ca.crt\" | base64decode}}" -n hono > ${TRUSTSTORE_PATH}
+kubectl get configmaps eclipse-hono-example-trust-store --template="{{index .data \"ca.crt\"}}" -n hono > ${TRUSTSTORE_PATH}
```

Comment:
FYI: The change in addressing the `ca.crt` for the `$TRUSTSTORE_PATH` via
`kubectl get configmaps eclipse-hono-example-trust-store`
instead of
`kubectl get secrets eclipse-hono-kafka-example-keys`
means that the Hono chart 
`example/certs/trusted-certs.pem`
instead of
`example/certs/ca-cert.pem`
is used.

Without this change, `mosquitto_pub/sub` will return an error:
`OpenSSL Error[0]: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed`

It seems that the `example/certs/root-cert.pem` included in `example/certs/trusted-certs.pem` is needed.